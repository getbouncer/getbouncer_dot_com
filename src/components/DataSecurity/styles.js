import styled from '@emotion/styled'

export const Icon = styled.img`
  width: 72px;
`

export const Headline = styled.h2`
  font-size: 2.857rem;
  margin: 28px 0 35px;
`

export const Text = styled.div`
  margin: 0 auto 30px;
  max-width: 676px;
`
