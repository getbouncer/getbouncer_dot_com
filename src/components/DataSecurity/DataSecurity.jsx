import React from 'react'
import { css } from '@emotion/core'
import { breakpoints } from '~styles'
import { Icon, Headline, Text } from './styles'
import dataIcon from '~images/icon-data.svg'
import dataBg from '~images/bg-data.png'

const DataSecurity = () => {
  const backgroundStyles = css`
    background-image: url(${dataBg});
    background-position: center center;
    background-repeat: no-repeat;
    background-size: cover;
    color: #fff;
    padding: 120px 0;
    text-align: center;

    @media ${breakpoints.smartphone} {
      padding: 60px 32px 68px;
    }
  `

  return (
    <section css={backgroundStyles}>
      <Icon src={dataIcon} alt="Data icon" />
      <Headline>Data handling and security</Headline>
      <Text>
        Bouncer Card is designed to keep user data safe and secure, and separates PCI data handling outside of Bouncer’s
        servers. We do analysis locally on the user’s mobile phone, and send back a subset of model output data that is
        stripped of image and PAN. This approach is PCI and GDPR ready.
      </Text>
    </section>
  )
}

export default DataSecurity
