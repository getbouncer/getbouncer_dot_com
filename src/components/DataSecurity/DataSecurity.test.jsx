import React from 'react'
import renderer from 'react-test-renderer'
import { describe, expect, it } from 'jest'
import DataSecurity from './DataSecurity'

describe('Data Security', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<DataSecurity />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
