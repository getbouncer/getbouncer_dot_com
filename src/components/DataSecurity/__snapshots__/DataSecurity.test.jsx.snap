// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data Security renders correctly 1`] = `
<section
  css={
    Object {
      "map": undefined,
      "name": "2n60pf",
      "next": undefined,
      "styles": "
    background-image: url(test-file-stub);
    background-position: center center;
    background-repeat: no-repeat;
    background-size: cover;
    color: #fff;
    padding: 120px 0;
    text-align: center;

    @media (max-width: 920px) {
      padding: 60px 32px 68px;
    }
  ",
      "toString": [Function],
    }
  }
>
  <img
    alt="Data icon"
    className="css-1y3nu44"
    src="test-file-stub"
  />
  <h2
    className="css-jfw3ak"
  >
    Data handling and security
  </h2>
  <div
    className="css-pm9scx"
  >
    Bouncer Card is designed to keep user data safe and secure, and separates PCI data handling outside of Bouncer’s servers. We do analysis locally on the user’s mobile phone, and send back a subset of model output data that is stripped of image and PAN. This approach is PCI and GDPR ready.
  </div>
</section>
`;
